src/Workspaces/Core/Portable/PatternMatching/PatternMatcher.cs

```diff
@@ -257,10 +257,7 @@ private static bool ContainsUpperCaseLetter(string pattern)
             // only after we've tried all other forms first.  This is the weakest of all matches.  For example, if
             // user types 'bin' we want to match 'OperatorBinary' (start of word) or 'BinaryInformationNode' (camel
             // humps) before matching 'Combine'.
-            // 
-            // We only do this for strings longer than three characters to avoid too many false positives when the
-            // user has only barely started writing a word.
-            if (patternIsLowercase && caseInsensitiveIndex > 0 && patternChunk.Text.Length >= 3)
+            if (patternIsLowercase && caseInsensitiveIndex > 0)
```

Comment:
That example would be matched even without this change.

But still, I think would be a correct thing to do since we shouldn't make these kinds of assumptions when user is writing in all lowercase. Personally, I often write in all lowercase without thinking about camel case parts, so it would definitely be annoying if intellisense didn't pick it up.

Reply:
the main issue here is that we have tried similar things int he past and it's been annoying in the opposite direction :)

Reply:
note: my exception will go away if this is just hte `>= 2` change.